scripts/Modules/infodisplay.ini

```diff
@@ -0,0 +1,35 @@
+[DEBUG]
```

Comment:
It would be nice if the config file that is used is not tracked by git.

Reply:
To clarify, this file should ABSOLUTELY be committed. Because we always want to be able to provide it alongside the scripts (We don't expect the user to have to create this file manually themselves). However, we will want to ignore any changes to the file. Is this doable?

Reply:
It would work if this file was the default configuration and is used to supply configuration the main config file lacks.